.github/workflows/flutter_markdown.yaml

```diff
@@ -0,0 +1,58 @@
+# Run a smoke test against package:flutter_markdown.
+
+name: flutter_markdown
+
+on:
+  schedule:
+    # “At 00:00 (UTC) on Sunday.”
+    - cron: '0 0 * * 0'
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions: read-all
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: clone dart-lang/markdown
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          repository: dart-lang/markdown
+          path: markdown
+
+      - name: clone flutter/packages
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          repository: flutter/packages
+          path: flutter_packages
+
+      - name: get flutter
+        uses: subosito/flutter-action@dbf1fa04f4d2e52c33185153d06cdb5443aa189d
```

Comment:
Can you say what this is?

Reply:
Installs the Flutter SDK?

Reply:
Haha I get that. I just don't know what "subosito" is .

Reply:
It seems to be the standard flutter 'install sdk' action; I'll add a comment here however.